src/us/kbase/workspace/database/provenance/CreditMetadata.java

```diff
@@ -222,14 +253,15 @@ public boolean equals(Object obj) {
 	 */
 	public static Builder getBuilder(
 			final String identifier,
+			final Organization repository,
```

Comment:
Is it guaranteed that organization information will always be available? This seems to imply that an individual person can't get credit for something

Reply:
Yep, it will be for all our data.

Reply:
Isn't that a problem? If I upload something to KBase as an independent researcher I want credit

Reply:
`repository` refers to the website/repo/organization from which the resource can be downloaded -- e.g. KBase, IMG, Dryad, figshare, etc. If the independent researcher stores their data on GitHub, that would be what would be captured in this field.

Reply:
: